Libraries/ActionSheetIOS/RCTActionSheetManager.m

```diff
@@ -170,7 +171,7 @@ - (CGRect)sourceRectInView:(UIView *)sourceView
                                            options:(NSDataReadingOptions)0
                                              error:&error];
       if (!data) {
-        failureCallback(error);
+        reject([error domain], [error localizedDescription], error);
```

Comment:
style: use property syntax (.) instead of method call for error.domain and error.localizedDescription when Apple documents them as properties:

<img width="243" alt="screenshot 2016-07-17 01 20 50" src="https://cloud.githubusercontent.com/assets/379606/16899544/d35ce68a-4bbc-11e6-8776-731998c7b17a.png">